doc/data/messages/i/import-error/bad.py

```diff
@@ -0,0 +1,2 @@
+# you didn't install it via pip, setup.py, wheel

```

Comment:
```suggestion
```
You can add comment in details.rst. sorry I can't suggest it as the file was deleted.

Reply:
Shifted the comments to details.rst.